src/serac/numerics/functional/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 #
 # SPDX-License-Identifier: (BSD-3-Clause)
 
-set(functional_depends mfem mpi serac_infrastructure serac_mesh serac_numerics)
+set(functional_depends serac_mesh serac_numerics)
```

Comment:
I find the directory structure here a bit odd. I feel that subdirectories should children of their parents. If `serac_functional` depends on `serac_numerics` should it be pulled out into its own directory or maybe have this directory moved into a `numerics_core` or something?

Reply:
except tests.... they should be under the thing they are testing.